### ISSUE TITLE
Code updation

### ISSUE THREAD

**Issue by hackintoshrao** [Kredaro/Deep-Learning-Training#21]
The example code of [Tensorflow with RELU and Dropout](https://github.com/Kredoai/Deep-Learning-Training/tree/master/Tensorflow/Deep_Neural_Nets_Dropout_RELU) has to be updated both in the notebook and in the Readme.
